test/common/testinsp/network.py

```diff
@@ -0,0 +1,52 @@
+from testinsp.base import TestInspector
```

Comment:
This also has never been doubtful. We know which interfaces are set up at any point in the test, and screenshots tell us their actual state.